pages/product.js

```diff
@@ -2,29 +2,11 @@ const { I } = inject();
 
 module.exports = {
   priceText: { xpath: '//div[@class="price"]/span' },
-  dropdownElement: { xpath: "//a[contains(@id, 'sbSelector')]" },
-  color: { xpath: `//ul[contains(@id, "sbOptions")]//a[contains(.,"Yellow")]` },
-
 
   async getProductPrice() {
     let product = await I.grabTextFrom(this.priceText);
     let productResult = I.getFloat(product);
     return parseFloat(productResult);
```

Comment:
move `parseFloat()` to `getFloat()` in `steps_file.js`.
Remove variables `product` and `productResult`. By the way, they should be named more specifically.
`return I.getFloat(await I.grabTextFrom(this.priceText));`

Reply:
> move `parseFloat()` to `getFloat()` in `steps_file.js`. Remove variables `product` and `productResult`. By the way, they should be named more specifically. `return I.getFloat(await I.grabTextFrom(this.priceText));`


It is my second code in life. I don’t know how to write it, and you never explained even though I wrote you on Skype. Here you see my version.

Reply:
here is what you have in `steps_file`:
![image](https://user-images.githubusercontent.com/19595492/209589025-fc17dc9b-d93c-415a-8705-7244e3a8edac.png)
here is what you can do:
![image](https://user-images.githubusercontent.com/19595492/209588992-8b2aff83-392a-4d3f-b284-78c362b37100.png)
so code in `product.js` and other places where price is grabbed is changed from:
![image](https://user-images.githubusercontent.com/19595492/209589055-6e093af3-2721-4d10-9d77-8e2856f15ada.png)
to:
![image](https://user-images.githubusercontent.com/19595492/209589089-70a057e7-9162-4aa7-ad8b-9e6c25cf9158.png)